src/test/java/org/folio/rest/impl/UsersAPIIT.java

```diff
@@ -60,6 +69,13 @@ static void beforeAll(Vertx vertx, VertxTestContext context) {
     final var okapiUrl = new OkapiUrl("http://localhost:" + port);
     final var headers = new OkapiHeaders(okapiUrl, tenant, token);
 
+    kafkaCluster = EmbeddedKafkaCluster.provisionWith(defaultClusterConfig());
+    kafkaCluster.start();
+    String[] hostAndPort = kafkaCluster.getBrokerList().split(":");
+    System.setProperty(KAFKA_HOST, hostAndPort[0]);
+    System.setProperty(KAFKA_PORT, hostAndPort[1]);
+    System.setProperty(KAFKA_ENV, KAFKA_ENV_VALUE);
```

Comment:
Seeing these lines are repeated quite a bit in the tests. Maybe move them out to a static helper class to make it more DRY.

Reply:
Moved to another class